WordPress/Sniffs/PHP/EnqueuedCheckSniff.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Sniffs\PHP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Based off the StrictInArraySniff.php Sniff, THis checks the Enqueued 4th Parameter to make sure a Version is available.
+ * The Enqueued functions are:
+ * wp_register_script
+ * wp_enqueue_script
+ * wp_register_style
+ * wp_enqueue_style
+ *
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1146
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.15.0 This new sniff will check for a version in an Enqueued WP function
+ */
+class EnqueuedCheckSniff extends AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.11.0
+	 * @var string
+	 */
+	protected $group_name = 'Enqueued';
+
+	/**
+	 * List of array functions to which a $strict parameter can be passed.
+	 *
+	 * The $strict parameter is the third and last parameter for each of these functions.
+	 *
+	 * The array_keys() function only requires the $strict parameter when the optional
+	 * second parameter $search has been set.
+	 *
+	 * @link http://php.net/in-array
+	 * @link http://php.net/array-search
+	 * @link http://php.net/array-keys
+	 *
+	 * @since 0.10.0
+	 * @since 0.11.0 Renamed from $array_functions to $target_functions.
+	 *
+	 * @var array <string function_name> => <bool always needed ?>
+	 */
+	
+	/**
+	 * List of Enqueued functions that need to be check to make sure
+	 * IF a source ($src) value is passed, then version ($ver) needs to have a value.
```

Comment:
This line and the two below it should be part of the class docblock. They are not directly related to this parameter so this info should be placed here.